### PR TITLE
Edit latest additions to fullnode doc

### PR DESCRIPTION
**Pull request by Clay-Mysten**

### CODE REVIEW

File: doc/src/build/fullnode.md [hunk 1]
```diff
@@ -82,42 +82,42 @@ transaction rate, etc) although we don't anticipate running a fullnode on
...
 
 0. *Prerequisite* Before beginning ensure that the following tools are
    installed in your environment:
     - Rust toolchain managed by [rustup](https://rustup.rs/)
     - `git`
     - `cmake`
 
-1. Setup your fork of the Sui repository.
-    - Go to the [Sui repository](https://github.com/MystenLabs/sui) on Github
+1. Set up your fork of the Sui repository:
```
Comment (bmwill) [inline doc/src/build/fullnode.md:99]: Setup is a word

File: doc/src/build/fullnode.md [hunk 2]
```diff
@@ -82,42 +82,42 @@ transaction rate, etc) although we don't anticipate running a fullnode on
...
     ```
     $ git clone https://github.com/<YOUR GITHUB USERNAME>/sui.git
     ```
-2. `cd` into your `sui` repository.
+2. `cd` into your `sui` repository:
     ```
     $ cd sui
     ```
-3. Setup the Sui repository as a git remote
+3. Set up the Sui repository as a git remote:
```
Comment (bmwill) [inline doc/src/build/fullnode.md:111]: Setup is a word

File: doc/src/build/fullnode.md [hunk 3]
```diff
@@ -82,42 +82,42 @@ transaction rate, etc) although we don't anticipate running a fullnode on
...
     ```
     $ git remote add upstream https://github.com/MystenLabs/sui
     $ git fetch upstream
     ```
-4. Checkout the devnet branch
+4. Check out the 'devnet' branch:
```
Comment (bmwill) [inline doc/src/build/fullnode.md:116]: Checkout is a word

File: doc/src/build/fullnode.md [hunk 4]
```diff
@@ -142,56 +142,55 @@ in how a fullnode is run.
...
+## Updating your fullnode with new releases
 
-Whenever a new release is deployed to `devnet` the blockchain state is
-generally wiped. In order to have your fullnode continue to properly
-synchronize with the new state of devnet you'll need to follow a few steps
-based on how you originally setup your node.
+Whenever a new release is deployed to `devnet`, the blockchain state is
+generally wiped clean. In order to have your fullnode continue to properly
+synchronize with the new state of devnet, you'll need to follow a few steps
+based on how you originally set up your node. See below.
```
Comment (bmwill) [inline doc/src/build/fullnode.md:177]: Setup is a word

File: doc/src/build/fullnode.md [hunk 5]
```diff
@@ -142,56 +142,55 @@ in how a fullnode is run.
...
+### Built from source
 
 If you followed the [Building from
-Source](#markdown-header-building-from-source) directions you can do the
-following to update:
+Source](#markdown-header-building-from-source) directions, update as follows:
 
-1. Shutdown your currently running fullnode.
-2. `cd` into where your local Sui repository is.
+1. Shut down your currently running fullnode.
```
Comment (bmwill) [inline doc/src/build/fullnode.md:184]: Shutdown is a word

File: doc/src/build/fullnode.md [hunk 6]
```diff
@@ -48,7 +48,7 @@ Today this synchronization process is performed by:
 Today this synchronization process is far from ideal as it requires listening
 to at a minimum 2f+1 validators to ensure that a fullnode has properly seen all
 new transactions. Overtime we will improve this process (e.g. with the
```
Comment (AllyMedina1) [inline doc/src/build/fullnode.md:50]: Overtime should be Over time

**Review by bmwill — APPROVED** [review]

**Pull request merged**